README.md

```diff
@@ -42,27 +42,25 @@ Ballerina as a JMS Producer
     }
     
     function jmsSender() {
-        // We define the connection properties as a map. 'providerUrl' or 'configFilePath' and the 'initialContextFactory' vary according to the JMS provider you use.
-        // In this example we connect to the WSO2 MB server.
-        jms:ClientConnector jmsEP;
-        jms:ConnectorProperties conProperties = {
-                             initialContextFactory:"wso2mbInitialContextFactory",
-                             configFilePath:"../jndi.properties",
-                             connectionFactoryName: "QueueConnectionFactory",
-                             connectionFactoryType : "queue"};
-        
-        // Create the JMS client Connector using the connection properties we defined earlier.
-        jmsEP = create jms:ClientConnector(conProperties);
-        
+        endpoint<jms:JmsClient> jmsEP {create jms:JmsClient (getConnectorConfig());}
```

Comment:
can we break this to multiple lines? like below
endpoint<jms:JmsClient> jmsEP {
           create jms:JmsClient (getConnectorConfig());
}